### DISCUSSION TITLE
When will tauri 2.0 be officially released?

### DISCUSSION THREAD

**Discussion by zwl1619** [tauri-apps/tauri#6734] [General]
When will tauri 2.0 be officially released? Will it support IOS and Android?

![Snipaste_2023-04-18_16-58-17](https://user-images.githubusercontent.com/13177997/232727488-9beb5a70-f867-4308-b694-f3a76f5c369b.png)

**Comment by FabianLars** [comment]
> When will tauri 2.0 be officially released?

We don't know yet. But we're at least trying to get it to beta this year afaik.

> Will it support IOS and Android?

Yes.

**Comment by liudonghua123** [comment]
The beta version is released, current the latest version is [tauri v2.0.0-beta.8](https://github.com/tauri-apps/tauri/releases/tag/tauri-v2.0.0-beta.8). Can't wait to use the latest v2.0.0 release.

**Comment by gabenddos** [comment]
I'm so excited to try out the tauri to build my cross platform app.
